File: Python/7kyu/0009putLetterInColumn/plc.py
def main():
    def build_row_text(index, character):
        result = "|"
        column = 0
        while column < 9:
            if column == index:
                result += character + "|"
            else:
                result += " |"
            column += 1
        print(result)
        return result
    build_row_text(3, "A")


if __name__ == "__main__":
    main()

# Status: Solved

File: JavaScript/7kyu/0075putLetterInColumn/plc.js
function buildRowText(index, character) {
  let result = "|"
  for (let divider = 1, column = 0; divider < 10; divider++, column++) {
    if (column === index) {
      result = result + character;
    } else {
      result = result + " ";
    }
    result = result + "|";
  }
  console.log(result);
  return result;
}

buildRowText(2, "A");

// Status: Solved